.changeset/green-buttons-reflect.md

```diff
@@ -0,0 +1,5 @@
+---
+'astro': minor
```

Comment:
Even though this is a new feature it's not a breaking change and since we are <1 I think we should mark this as a patch.